__tests__/app.test.js

```diff
@@ -27,75 +27,90 @@ describe("GET /api", () => {
   });
 });
 
-
 describe("GET /api/topics", () => {
   test("gives status of 200 and responds with an object of topics", () => {
     return request(app)
       .get("/api/topics")
       .expect(200)
       .then(({ body: { allTopics } }) => {
         allTopics.forEach((topic) => {
-					expect(Object.keys(topic)).toHaveLength(2);
-					expect(topic).toMatchObject({
-						slug: expect.any(String),
-						description: expect.any(String),
-					});
-				});
-        
+          expect(Object.keys(topic)).toHaveLength(2);
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
       });
   });
 });
 
-
-
 describe("GET /api/articles/:article_id", () => {
   test("gives status of 200 and responds with the appropriate article", () => {
     return request(app)
       .get("/api/articles/2")
       .expect(200)
       .then(({ body: { chosenArticle } }) => {
         expect(Object.keys(chosenArticle)).toHaveLength(8);
-        expect(chosenArticle).toMatchObject(
-          { 
-            article_id: expect.any(Number),
-            title: expect.any(String),
-            topic: expect.any(String),
-            author: expect.any(String),
-            body: expect.any(String),
-            created_at: expect.any(String),
-            votes: expect.any(Number),
-            article_img_url: expect.any(String),
-          }
-        )
-        
+        expect(chosenArticle).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
       });
   });
 });
 
-
 describe("GET /api/articles", () => {
   test("gives status of 200 and responds with all articles", () => {
     return request(app)
       .get("/api/articles")
       .expect(200)
       .then(({ body: { allArticles } }) => {
         allArticles.forEach((article) => {
-					expect(Object.keys(article)).toHaveLength(8);
-					expect(article).toMatchObject({
-						article_id: expect.any(Number),
+          expect(Object.keys(article)).toHaveLength(8);
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
             title: expect.any(String),
             topic: expect.any(String),
             author: expect.any(String),
             created_at: expect.any(String),
             votes: expect.any(Number),
             article_img_url: expect.any(String),
             comment_count: expect.any(Number),
-					});
-				});
-        
-        
-        expect(allArticles).toBeSortedBy('created_at', { descending: true });
+          });
+        });
+
```

Comment:
I think this sprint instruction suggests the body shouldn't be on the returned object it could be an unnecessarily large data request that wouldn't be used. `.toMatchObject` is perfectly fine, but is there an additional way to test the body key is not being returned also?